sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -889,4 +889,18 @@ class AdaptiveQueryExecSuite
       }
     }
   }
+
+  test("SPARK-32753: Only copy tags to node with no tags") {
+    withSQLConf(
+      SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true"
+    ) {
```

Comment:
Indentation?
```scala
withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> "true") {
```

Reply:
Sure. BTW, if this indentation style must be followed, should it be added to checkstyle test ?